globals.go

```diff
@@ -329,6 +335,15 @@ var (
 		PlannerVersionTunable,
 	}
 
+	// ValidBootstrapMethods is the set of of valid methods to bootstrap the
+	// host.
+	ValidBootstrapMethods = []string{
```

Comment:
I might make a "validate" method rather than have a mutable global variable. I know the precedent isn't great here.

Reply:
I put the validation method in `model/distro/distro.go`.